src/modules/drawing/components/DrawingStylePopup.vue

```diff
@@ -27,6 +27,28 @@
                     <button type="button" class="btn btn-default" @click="onDelete">
                         <font-awesome-icon :icon="['far', 'trash-alt']" />
                     </button>
+                    <button
+                        v-if="feature.get('type') === 'MARKER' || feature.get('type') === 'TEXT'"
+                        ref="textStylePopoverBtn"
+                        type="button"
+                        class="btn btn-default text-style"
+                    >
+                        <font-awesome-icon :icon="['fas', 'font']" />
+                    </button>
+                </div>
+                <div
+                    v-if="feature.get('type') === 'MARKER' || feature.get('type') === 'TEXT'"
```

Comment:
it could even make more sense to have two computed properties called `isFeatureMarker` and `isFeatureText` as that's the only thing we use `feature.get('type')` for.